### DISCUSSION TITLE
.text-sm ul.bullet --> not working

### DISCUSSION THREAD

**Discussion by kiliandr** [tailwindlabs/tailwindcss#11889] [Help]
Hi,
i added the following code to the input.css

`@tailwind base;
@tailwind components;
@tailwind utilities;

ul.bullet {
	@apply list-none;
	@apply list-outside;
	@apply ml-8;
	@apply my-4;
}

ul.bullet li {
	@apply mt-4;
	@apply relative;
}

ul.bullet li::before {
	content:'';
	@apply absolute;
	@apply -left-8;
	@apply top-1.5;
	@apply bg-secondary-600;
	@apply w-3;
	@apply h-3;
}

.text-sm ul.bullet li::before { /* in Text mit grossem Text mehr Abstand nach oben */
	@apply top-1;
}`

But the top-1-class is also attached to the ul.bullet (outside of .text-sm).
Isnt all css code outside a layer used like normal css?

**Comment by wongjn** [comment]
I tried your code in a [Tailwind Play](https://play.tailwindcss.com/J8f2f6l2f3) and I was unable to replicate your issue. Consider providing a [Tailwind Play](https://play.tailwindcss.com/) that reproduces the unexpected behavior.

**Comment by kiliandr** [comment]
Hi,
it seems that it is a problem with the css in general...

https://codepen.io/kili123/pen/gOZaxOz

The :read-only-Selector is also applied to the <select>-Element that doesnt have an readonly-attribute...

**Reply by tlgreg** [reply]
This is something different though.

By spec `:read-only` selects anything that is not editable.
"Editable" is only `<input>` and `<textarea>` that are not `disabled` or `readonly`. Plus also anything that is `contenteditable`.
If you change the selector in the pen to `*:read-only` you will see that it will apply it to the whole document.

[:read-only (MDN)](https://developer.mozilla.org/en-US/docs/Web/CSS/:read-only)
